Sources/Target/Darwin/Thread.cpp

```diff
@@ -33,8 +32,39 @@ namespace Darwin {
 
 using Host::Darwin::LibProc;
 
-Thread::Thread(Process *process, ThreadId tid)
-    : super(process, tid), _lastSyscallNumber(-1) {}
+Thread::Thread(ds2::Target::Process *process, ThreadId tid)
+    : super(process, tid), _lastSyscallNumber(-1) {
+  //
+  // Initially the thread is stopped.
+  //
+  _state = kStopped;
```

Comment:
The body of this constructor is useless. _state is already set to kStopped by `POSIX::Thread::Thread` and `_lastSyscallNumber` is already set to `-1` by the initializer list.